vscode/package.json

```diff
@@ -397,7 +397,12 @@
           }
         }
       }
-    ]
+    ],
+    "configurationDefaults": {
+      "[qsharp]": {
+        "debug.saveBeforeStart": "none"
```

Comment:
Instead of making this configurable, I think we can pass in `suppressSaveBeforeStart` with the `DebugSessionOptions` object when we call `startDebugging`. I tend to think that would be simpler, what do you think?

Reply:
Either way. I think by doing it here it allows the user to set it if they'd prefer to save on launch (similar to have we have `"editor.formatOnSave": true` in our .vscode settings for some languages. If we set it in the session, the user can't override.

Reply:
RE the TODO: Plan was to fix that before I'd merge, just wanted to get the PR up for feedback last night. Will fix in this PR before merging.